projects/batfish/src/main/java/org/batfish/symbolic/bdd/BDDRoute.java

```diff
@@ -52,19 +51,13 @@
     allProtos.add(Protocol.OSPF);
     allProtos.add(Protocol.BGP);
 
-    CallbackHandler handler = new CallbackHandler();
```

Comment:
The JavaBDD library prints garbage collection information to stdout by default. The only way to suppress this printing is to register a callback method using reflection, which is what this is doing. I think the better solution if you are getting an unused variable warning, would be to uncomment the registerGCCallback line to have no printing by default.